mesonbuild/interpreter/dependencyfallbacks.py

```diff
@@ -351,8 +354,14 @@ def lookup(self, kwargs: TYPE_nkwargs, force_fallback: bool = False) -> Dependen
         # Try all candidates, only the last one is really required.
         last = len(candidates) - 1
         for i, item in enumerate(candidates):
-            func, func_args, func_kwargs = item
-            func_kwargs['required'] = required and (i == last)
+            func, func_args = item
+            func_kwargs: DoSubproject = {
+                'required': required and (i == last),
+                'cmake_options': [],
+                'default_options': self.default_options,
+                'options': None,
+                'version': [],
+            }
             kwargs['required'] = required and (i == last)
```

Comment:
```suggestion
            kwargs['required'] = func_kwargs['required']
```

(or vice versa, swap the two assignments so that you can use `'required': kwargs['required']` in `func_kwargs`)